tests/queries.py

```diff
@@ -168,6 +168,22 @@ def poll_query_message(indx, namespace, client_po_name, fields, findFails=False,
 
 def query_message(indx, namespace, client_po_name, fields, find_fails=False, start_time=None, queries=None,
                   is_print=True):
+    """
+    Send a query and return its' hits, this function is used by most of the query functions.
```

Comment:
```suggestion
    Send a query and return its hits, this function is used by most of the query functions.
```

Reply:
fixed in 88127f439bd8e20b30ac26cb0f739a8059935186